pkg/lightning/restore/table_restore.go

```diff
@@ -694,15 +693,28 @@ func (tr *TableRestore) postProcess(
 					return false, err
 				}
 				if !needChecksum {
+					if rc.cfg.TikvImporter.DuplicateDetection {
+						if err := rc.backend.CollectLocalDuplicateRows(ctx, tr.encTable, rc.cfg.TiDB.SQLMode); err != nil {
+							tr.logger.Error("collect local duplicate keys failed", log.ShortError(err))
```

Comment:
Is `CollectLocalDuplicateRows` is called before when `needChecksum` is `true`?

Reply:
Yes. We do not need to checksum when it is not the lastest lightning which finishes task.

Reply:
It seems CollectLocalDuplicateRows is not called when needChecksum is true.